### DISCUSSION TITLE
the issue with Next JS and Tailwind has followed the guide on Tailwind but Tailwind doesn't render

### DISCUSSION THREAD

**Discussion by ahmadsoni** [tailwindlabs/tailwindcss#10069] [Help] [answered]
I use Next JS Typescript and Tailwind but when I try the tutorial on the tailwind website with next , tailwind doesn't work properly and can't render.

I tried to provide a solution with a CDN and it worked but it doesn't match the project I'm running

do Next JS and Tailwind have Issues?

`/**
 * @type {import('@types/tailwindcss/tailwind-config').TailwindConfig}
 */
module.exports = {
	content: [
		'./pages/**/*.{ts,tsx}',
		'./public/**/*.html',
	],
	plugins: [
		require('flowbite/plugin'),
	],
	theme: {},
};
`

**Comment by ProPhycient5** [comment, marked as answer]
It should work. 

Please do replace `content` with below one. Check if it is working.

```
  content: [
    "./pages/**/*.{js,ts,jsx,tsx}",
    "./components/**/*.{js,ts,jsx,tsx}",
  ]
  ```

**Reply by ahmadsoni** [reply]
thank you, it turns out that you really have to follow the best practice on the official website

**Reply by ProPhycient5** [reply]
You are welcome🙏🏼, @adambrgmn. Yes, these are all mentioned on the website. Please opt for my reply/answer as `Answer` and close this issue, if it resolves your problem.

**Reply by ProPhycient5** [reply]
> thank you, it turns out that you really have to follow the best practice on the official website

Please close the discussion by marking the above reply as `Answer`.
